### PR TITLE
Update "submit review" and "request review" actions to work for projectsV2

### PR DESCRIPTION
**Pull request by ilia-kebets-sonarsource**
When merged, we would need to change the branches to @V1 like:

```yaml
      - uses: sonarsource/gh-action-lt-backlog/MoveCardToReview@V1
```

instead of 

```yaml
      - uses: sonarsource/gh-action-lt-backlog/MoveCardToReview@ use-until-merged
```

### CODE REVIEW

**Review by victor-diez-sonarsource — APPROVED** [review]

File: .github/workflows/RequestReview.yml [hunk 1]
```diff
@@ -8,10 +8,20 @@ jobs:
   MoveCardToReview_job:
     name: Move card to review
     runs-on: ubuntu-latest
+    permissions:
+      id-token: write
     # PRs from forks don't have required token authorization
-    if: github.event.pull_request.head.repo.full_name == github.repository
+    if: |
+        github.event.pull_request.head.repo.full_name == github.repository
+        && github.event.review.author_association != 'NONE'
```
Comment (yassin-kammoun-sonarsource) [inline .github/workflows/RequestReview.yml:16]: What's the meaning of this extra condition ? What's an author association ?

Comment (ilia-kebets-sonarsource) [reply .github/workflows/RequestReview.yml:16]: it means that the reviewer has at least some rights on the repo: https://docs.github.com/en/webhooks-and-events/events/issue-event-types#properties-for-commented

Comment (yassin-kammoun-sonarsource) [reply .github/workflows/RequestReview.yml:16]: 👍

Comment (yassin-kammoun-sonarsource) [reply .github/workflows/RequestReview.yml:16]: Do you plan to document this kind of detail somewhere?

Comment (ilia-kebets-sonarsource) [reply .github/workflows/RequestReview.yml:16]: These actions are made to be copy-pasted from their templates, and a few parameters replaced like this one from here: https://github.com/SonarSource/gh-action-lt-backlog/tree/master/MoveCardAfterReview#projects-v2

So we don't deem the documentation necessary.

**Pull request merged**